app/scripts/views/complete_sign_up.js

```diff
@@ -62,7 +63,7 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       return self.fxaClient.verifyCode(uid, code)
           .then(function () {
             self.logScreenEvent('verification.success');
-            return self.broker.afterCompleteSignUp(self.getAccount());
+            return self.broker.afterCompleteSignUp(self.getAccount(), self.relier.get('alwaysRedirect'));
```

Comment:
Again, I would not pass in self.relier.get('alwaysRedirect'), but let the broker fetch this value from the relier directly.